Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1604,7 +1604,7 @@ private string GetCommandLineArguments(string outputPath, StreamState state, boo
                 Path.GetFileNameWithoutExtension(outputPath));
         }
 
-        var hlsArguments = GetHlsArguments(isEventPlaylist, state.SegmentLength);
+        var hlsArguments = string.Format(CultureInfo.InvariantCulture, "-hls_playlist_type {0} -hls_list_size 0", isEventPlaylist ? "event" : "vod");
```

Comment:
string interpolation is preferred (I don't remember if the parentheses are required for ternarys inside string interpolation)
```suggestion
        var hlsArguments = $"-hls_playlist_type {(isEventPlaylist ? "event" : "vod")} -hls_list_size 0"; 
```

Reply:
Tested and it works. Parentheses is necessary here.